tox.ini

```
@@ -83,13 +83,19 @@ commands =
     !headless: python -m pytest {env:PYTEST_PATH:} --color=yes --basetemp={envtmpdir} \
         --cov-report=xml --cov={env:PYTEST_PATH:napari} --ignore tools --maxfail=5 \
         --json-report --json-report-file={toxinidir}/report-{envname}.json {posargs}
+        --skip_examples
+
     # do not add ignores to this line just to make headless tests pass.
     # nothing outside of _qt or _vispy should require Qt or make_napari_viewer
     headless: python -m pytest --color=yes --basetemp={envtmpdir} --ignore napari/_vispy \
         --ignore napari/_qt --ignore napari/_tests --ignore tools \
         --json-report --json-report-file={toxinidir}/report-{envname}.json {posargs}
 
 
+[testenv:py{38,39,310}-{linux,macos,windows}-{pyqt5,pyside2}-examples]
+commands =
+    python -m pytest napari/_tests/test_examples.py --color=yes --basetemp={envtmpdir} {posargs}
```

```suggestion
    python -m pytest napari/_tests/test_examples.py -v --color=yes --basetemp={envtmpdir} {posargs}
```

(few enough that it's nice to see)